### PR TITLE
Adds status select field to complaint detail view

### PR DESCRIPTION
**Pull request by el-mapache**
[Link to ZenHub issue.](https://github.com/18F/crt-portal/issues/248)

## What does this change?

* Adds `select` element to complaint details view with available `status` values
* Extends ComplaintSelect widget to allow for custom labeling
* Styles will not match mocks yet

### Author

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] Check for [accessibility](/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

File: crt_portal/cts_forms/widgets.py [hunk 1]
```diff
@@ -23,6 +23,25 @@ class ComplaintSelect(ChoiceWidget):
...
+        return self.label
+
+    def render(self, name, value, attrs=None, renderer=None):
+        extra_context = {
+            'label': self.label_for_widget()
+        }
+        context = self.get_context(name, value, attrs)
+        context.update(extra_context)
+
+        return self._render(self.template_name, context, renderer)
```
Comment (el-mapache) [inline crt_portal/cts_forms/widgets.py]: I directly call render here, but it is probably better / safer to call super().render(...args)?

Comment (el-mapache) [reply crt_portal/cts_forms/widgets.py]: Calling super here seems to open a large can of worms, so I wonder if maybe this is ok for now 😬

**Comment by LindsayYoung** [comment]
Small thing- if you change the status or section, it doesn't preserve the link in the close button.

**Comment by el-mapache** [comment]
@LindsayYoung I've updated the code, but I wonder if there is a cleaner (or more Django-specific) way to achieve this logic?

**Comment by LindsayYoung** [comment]
> @LindsayYoung I've updated the code, but I wonder if there is a cleaner (or more Django-specific) way to achieve this logic?

I am thinking we can add a hidden field that passes the next variable.

**Comment by el-mapache** [comment]
works for me!

**Pull request merged**